tools/pythonpkg/tests/fast/pandas/test_df_object_resolution.py

```diff
@@ -329,6 +363,67 @@ def test_map_fallback_nullkey_coverage(self, pandas, duckdb_cursor):
         ):
             converted_col = duckdb_cursor.sql("select * from x").df()
 
+    @pytest.mark.parametrize('pandas', [NumpyPandas(), ArrowPandas()])
+    def test_structs_in_nested_types(self, pandas, duckdb_cursor):
```

Comment:
@Mytherin I added this test, it contains some behavior tests for incompatible struct schemas inside nested types